src/controller.rs

```diff
@@ -348,7 +348,7 @@ fn file_to_kind(path: &Path) -> &'static str {
     }
 }
 
-fn files_in_dir(path: &str, prefix: &Option<String>, skip: &[PathBuf]) -> Vec<PathBuf> {
+fn files_in_dir(path: &str, prefix: Option<&String>, skip: &[PathBuf]) -> Vec<PathBuf> {
```

Comment:
What about &str?

Reply:
Applied the suggested adjustment.
Seems more generic. Thank you.